src/errors/test-run/index.js

```diff
@@ -3,6 +3,7 @@
 // Do not use any browser or node-specific API!
 // -------------------------------------------------------------
 import { TEST_RUN_ERRORS } from '../types';
+import * as diff from '../../utils/diff/index';
```

Comment:
```suggestion
import * as diff from '../../utils/diff'
```

Reply:
There is a file in `/src/utils/` named `diff.js` so I'll change it to `../../utils/diff/`